homeassistant/components/utility_meter/sensor.py

```diff
@@ -655,6 +659,15 @@ def async_source_tracking(event):
 
         self.async_on_remove(async_at_started(self.hass, async_source_tracking))
 
+        @callback
+        def async_track_time_zone(event):
+            """Reconfigure Scheduler after time zone changes."""
+            self._config_scheduler()
```

Comment:
Do we want to avoid this if the time zone was not changed?

Reply:
I don't think it is necessary to avoid, the scheduler is already setup on every HA restart and it's a generator

Reply:
Sure, but time zone changes are very rare, other config changes are not.

Reply:
They would not influence the functioning of utility_meter, previously the scheduler was even updated every cycle

Reply:
I've added a check nonetheless